config/settings.yml

```diff
@@ -1001,6 +1002,8 @@
   :console_proxy_port:
     :start: 6000
     :end: 7000
+  :custom_brand: false
+  :custom_login_logo: false
```

Comment:
I notice this is added to the `:server` key as opposed to `:product`.  Are we expecting a different custom brand per server?

Reply:
What will these be used for?  I don't see them elsewhere in the PR.

Reply:
This is already being used and missing, I added this for the sake of consistency.

Reply:
I was not really thinking about the location, added the missing and the needed key to the same place where the existing `custom_logo` was.